### ISSUE TITLE
Open Preview shows wrong file in preview tab (markdown)

### ISSUE THREAD

**Issue by AArnott** [Microsoft/vscode#16464]
- VSCode Version: 1.7.2
- OS Version: Windows

Steps to Reproduce:

1. Open a Folder with several .md files
2. Open a few of the .md files
3. click Open Preview to view the rendered markdown

The preview shown often does not match the file that was active when the Open preview command was invoked.

I've never changed settings explicitly. But here are the files:

user settings.json:
```json
{
    "git.confirmSync": false
}
```
workspace settings.json:
```json
// Place your settings in this file to overwrite default and user settings.
{
}
```
![previewtab](https://cloud.githubusercontent.com/assets/3548/20874531/ac1ee6b2-ba67-11e6-8f24-261bb138e6a3.gif)

**Comment by chrmarti** [comment]
Cannot reproduce. Anything specific about your setup? Could you append your settings so we can check if these affect the behavior?

**Comment by AArnott** [comment]
I've added the data you requested and an animation to demonstrate how I repro the problem.

**Comment by mjbvz** [comment]
@AArnott I believe this was fixed by #16298 

The fix is available in the [latest insiders builds](https://code.visualstudio.com/insiders) and will be picked up in VSCode 1.8

Please let me know if you are still seeing this issue after upgrading.

Thanks

**Comment by AArnott** [comment]
I agree that dupe looks promising. thanks.
